mix.exs

```diff
@@ -1,14 +1,14 @@
 defmodule UeberauthGitlab.Mixfile do
   use Mix.Project
 
-  @version "0.2.0"
+  @version "0.3.0"
```

Comment:
Can you revert this back? It's good practice to leave the version unchanged until it is time to make a new release.